### ISSUE TITLE
[Feature Requests] Playlists & Repeat (+A-B)

### ISSUE THREAD

**Issue by RDusr05** [anilbeesetti/nextplayer#757]
Hello

I am missing the possibility to create a playlist for several videos.

It would also be nice to repeat videos or to run entire playlists in a continuous loop.

Also an A-B function to repeat certain sections of a video.

Could these functions be built into the app one day?

Many thanks for this great app!

**Comment by exercismnow** [comment]
Agreed! I think there's no easy to use FOSS android A-B repeat player. This would be a great feature for Next Player

To practice music or dance, you might have to use a closed-source player like RAV Player.

https://play.google.com/store/apps/details?id=com.toth.loopplayerii&hl=en-US

**Comment by Noitarud** [comment]
(ideally one topic per issue)

a-b is good for learning languages
